### PR TITLE
Updated team practices to include actionable support tickets

### PR DESCRIPTION
**Pull request by phirefly**

### CODE REVIEW

File: doc/team_practices.md [hunk 1]
```diff
@@ -212,3 +212,9 @@ If the bug is actionable, a story can be created for it and prioritized in the
 `Needs Grooming` for the next grooming session.
 
 
+## Support
+
+Support tickets for existing users are managed at [18F's User Voice account](https://18f.uservoice.com).
+The delivery team rotates assignment to respond to user issues and bug reports. As support requests are
+received, tickets are immediately actionable. The developer can assign the ticket to her/himself as
```
Comment (jessieay) [inline doc/team_practices.md]: `her/himself` -> `themself` (so we can be non gender binary)

Comment (jessieay) [inline doc/team_practices.md]: maybe we should include language that indicates it *should* be acted on immediately.

**Comment by pkarman** [comment]
+1 to @jessieay comments -- opened #896

**Comment by rememberlenny** [comment]
Updated with notes and merged.

**Pull request closed without merging**